bridge_adaptivity/module/policies/base.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models.aggregates import Count, Sum
+
+
+class BaseGradingPolicy(object):
```

Comment:
It looks like an interface if it is - use please ABCMeta

Reply:
done